mobile/screens/route_type_page.js

```diff
@@ -1,33 +1,80 @@
 
 const Route_type_page = () => {
     const nav = useNavigation();
 
     return (
         <View style={styles.layout}>
-            <Text style={styles.title}>Routes</Text>
-            <Pressable
-                onPress={() =>
-                    nav.navigate("RouteList", {
-                        apiCalled: "walkroutes",
-                        routeType: "Wandelroutes",
-                    })
-                }
-            >
-                <Text>Wandelroute</Text>
-            </Pressable>
-            <Pressable
-                onPress={() =>
-                    nav.navigate("RouteList", {
-                        apiCalled: "bikeroutes",
-                        routeType: "Fietsroutes",
-                    })
-                }
-            >
-                <Text>Fietsroute</Text>
-            </Pressable>
+            <View style={styles.innerLayout}>
+                <View style={styles.upperLayout}>
+                    <View style={styles.headerPage}>
+                        <Image
+                            source={require("../assets/logohighres.png")}
+                            style={{ marginTop: 25 }}
+                        />
+
+                        <View style={styles.inlineIconText}>
+                            <Text style={styles.title}>
+                                Kies een route type
+                            </Text>
+                        </View>
+                    </View>
+                </View>
+                <View style={type.buttons}>
+                    <View style={type.typeButton}>
+                        <Pressable
+                            onPress={() =>
+                                nav.navigate("RouteList", {
+                                    apiCalled: "bikeroutes",
+                                    routeType: "Fietsroutes",
+                                })
+                            }
+                            style={type.bikeButton}
+                        >
+                            <Ionicons
+                                name="bicycle"
+                                size={105}
+                                color="black"
+                                style={type.icon}
+                            />
+
+                            <Text style={type.buttonText}>Fietsroutes</Text>
+                        </Pressable>
+                    </View>
+                    <View style={type.typeButton}>
+                        <Pressable
+                            onPress={() =>
+                                nav.navigate("RouteList", {
+                                    apiCalled: "walkroutes",
+                                    routeType: "Wandelroutes",
+                                })
+                            }
+                            style={type.walkButton}
+                        >
+                            <FontAwesome5
+                                name="walking"
+                                size={100}
```

Comment:
Misschien deze iets kleiner maken, want de wandelaar is op het moment een stuk groter dan de fietser, hoeft niet perse maar staat wel beter imo